src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -24,7 +24,14 @@ def get_towncrier_config(
     """Return the towncrier config dictionary."""
     try:
         # Towncrier >= 19.9.0
-        return load_config_from_file(str(project_path), str(final_config_path))
+        config = load_config_from_file(
+            str(project_path), str(final_config_path),
+        )
     except TypeError:
         # Towncrier < 19.9.0
-        return load_config_from_file(str(final_config_path))
+        config = load_config_from_file(str(final_config_path))
```

Comment:
No need to rewrite this line. Exiting early is easier to follow.
```suggestion
        return load_config_from_file(str(final_config_path))
```